src/Integration.Vsix.UnitTests/Settings/GeneralOptionsDialogPageTests.cs

```diff
@@ -68,19 +72,69 @@ public void OnActivate_ControlsAreConfiguredFromSettings1()
             // Assert
             page.Control.Should().NotBeNull();
             page.Control.DaemonVerbosity.SelectedItem.Should().Be(DaemonLogLevel.Verbose);
+
+            // Daemon is not installed, so deactivate options should not be visible
+            page.Control.DeactivateButton.Visibility.Should().Be(Visibility.Collapsed);
+            page.Control.DeactivateText.Visibility.Should().Be(Visibility.Collapsed);
+            page.Control.VerbosityPanel.Visibility.Should().Be(Visibility.Collapsed);
+
+            // ... and activate options should be visible
+            page.Control.ActivateButton.Visibility.Should().Be(Visibility.Visible);
+            page.Control.ActivateText.Visibility.Should().Be(Visibility.Visible);
+            page.Control.ShowAdditionalLanguageDownloadDialogue.IsChecked.Should().BeFalse();
+            page.Control.ShowAdditionalLanguageDownloadText.Visibility.Should().Be(Visibility.Visible);
         }
 
+
         [TestMethod]
-        public void OnActivate_ControlsAreConfiguredFromSettings2()
+        public void OnActivate_WhenDaemonIsInstalled_ControlsAreConfiguredFromSettings1()
```

Comment:
It's not obvious in the GitHub diff view, but the next two tests already existed.
I've changed the names and added additional assertions to check the state of all of the controls.